File: ts/wam/5024.ts
export const WamStickerError = 5024 as const

export interface StickerErrorEvent {
  /** field 1, wire `sticker_error_type` */
  stickerErrorType?: STICKERERRORTYPE
  /** field 2, wire `sticker_message_type` */
  stickerMessageType?: STICKERSENDMESSAGETYPE
}

File: rust/wam/5024.rs
pub const WAM_STICKER_ERROR: u32 = 5024;

#[derive(Debug, Default)]
pub struct StickerErrorEvent {
    /// field 1, wire `sticker_error_type`
    pub sticker_error_type: Option<STICKERERRORTYPE>,
    /// field 2, wire `sticker_message_type`
    pub sticker_message_type: Option<STICKERSENDMESSAGETYPE>,
}